nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+
+const app = express();
+
+
+const addition=(req,res)=>{
+  const first=Number(req.query.first);
+  const second=Number(req.query.second)
+  const result=first+second;
+  if(('first' in req.query)&&('second' in req.query)){
+    return res.send(` ${result}`)
+    
+  }
+  return res.status(400).end()
+}
+
+
+const multiplication=(req,res)=>{
+  const first=Number(req.params.first);
```

Comment:
Using `Number` to convert to number is a bit unsafe. I would use parseInt instead!

Reply:
Thanks, changes implemented